### DISCUSSION TITLE
[Feature] Tagging a face without adding it to the training set

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#6001] [Feature Request]
### The feature

Hi, I'd love to be able to tag a person on an already recognized facebox, but making sure that it does not count for distance calculation or anything related to clustering for the future pictures.

Concrete exemple: my son wears a Hulk mask for Carnival. I don't want the picture to be classified with all the Hulk costumes in some random pictures, I'd like the picture to appear in my son's faces album. Yet, I don't want other Hulk costumes to be classified as my son in the future.

So the idea is just to add a searchable hashtag to the pic, but a special one, since it is still associated with the bounding box of the face on the photo. Therefore, if my son is wearing an Elsa costume and my daughter an Hulk costume, I can still mouseover and know who is who.

Current possible workaround:
- Untag the pic and add a textual description to the picture, so it can be indexed in search.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/2472#discussioncomment-6850486
